subprojects/packagefiles/kafel/meson.build

```diff
@@ -0,0 +1,72 @@
+project('kafel', 'c',
+  version: '2023.10.10',
+  default_options: [
+    'c_std=gnu11',
+  ])
+
+flex = find_program('flex')
+bison = find_program('bison')
+pkgconf = import('pkgconfig')
+
+run_command(flex, '--outfile=lexer.c', 'src/lexer.l', check: true)
+run_command(bison, '-o=parser.c', 'src/parser.y', check: true)
+
+kafel_sources = files(
+  'src/kafel.c',
+  'src/context.c',
+  'src/codegen.c',
+  'src/expression.c',
+  'src/includes.c',
+  'src/parser_types.c',
+  'src/policy.c',
+  'src/range_rules.c',
+  'src/syscall.c',
+  'src/syscalls/amd64_syscalls.c',
+  'src/syscalls/i386_syscalls.c',
+  'src/syscalls/aarch64_syscalls.c',
+  'src/syscalls/mipso32_syscalls.c',
+  'src/syscalls/mips64_syscalls.c',
+  'src/syscalls/riscv64_syscalls.c',
+  'src/syscalls/arm_syscalls.c',
+  'lexer.c',
+  'parser.c'
+)
+
+kafel_inc = include_directories('include', 'src')
+
+kafel_lib = library('kafel',
+  sources: kafel_sources,
+  include_directories: kafel_inc,
+  install: true
```

Comment:
```meson
  gnu_symbol_visibility : 'hidden',
  soversion : 1,
```
is also needed to match the makefile.

You'll need to double check that soversion though, and make sure it matches the makefile.

Reply:
Thank you, I fixed it!